webapp/channels/src/sass/layout/_markdown.scss

```diff
@@ -348,22 +348,19 @@ h6.markdown__heading {
     }
 }
 
-ol.markdown__list {
-    display: table;
-    padding: 0;
-    counter-reset: list;
-
+.markdown__list {
     >li {
-        display: table-row;
-        counter-increment: list;
-        word-break: break-word;
-
-        &::before {
-            display: table-cell;
-            padding-right: 8px;
-            content: counter(list) ".";
-            text-align: right;
-            word-break: keep-all;
+        &.list-item--task-list {
```

Comment:
It seems to me that the specific rules for markdown lists should go in this file, in order to be usable wherever markdown is possible (channel purpose, etc). I removed the hacks that existed to handle ordered lists, to get closer to github's approach.